java/core/src/test/java/org/bondlib/SomethingFloatTest.java

```diff
@@ -1,9 +1,10 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-package com.microsoft.bond;
+package org.bondlib;
 
-import com.microsoft.bond.helpers.FloatingPointHelper;
+import org.bondlib.helpers.FloatingPointHelper;
+import org.junit.Assert;
```

Comment:
The renaming has also adjusted imports here. Can we do that in a different commit?